lib/services/protocol_service.ts

```diff
@@ -1150,21 +1150,21 @@ export class SNProtocolService extends PureService implements EncryptionDelegate
     }
   }
 
-  /** 
+  /**
    * When a download-first sync completes, it means we've completed a (potentially multipage)
    * sync where we only downloaded what the server had before uploading anything. We will be
    * allowed to make local accomadations here before the server begins with the upload
    * part of the sync (automatically runs after download-first sync completes).
-   * We use this to see if the server has any default itemsKeys, and if so, allows us to 
+   * We use this to see if the server has any default itemsKeys, and if so, allows us to
    * delete any never-synced items keys we have here locally.
    */
   private async handleDownloadFirstSyncCompletion() {
     /**
     * Find items keys with null or epoch updated_at value, indicating
     * that they haven't been synced yet.
     */
-    const itemsKeys = this.itemsKeys;
-    const neverSynced = itemsKeys.filter((key) => {
+    const itemsKeys = this.itemsKeys();
```

Comment:
We probably don't need to make a variable out of this, as it's just being used once. This way would make it more consistent with how we access `this.itemsKeys()` inline below.

Reply:
It's used a second time on line 1174:
```
const defaultSyncedKey = itemsKeys.find((key) => { /* ... */ }
```

Reply:
Someone maintaining this code in the future might see the second this.itemsKeys() reference and think, oh, they forget they created a variable, and they'll swap out the function call for the variable. How do we prevent that 😅

Reply:
(Also no tests in place that would catch that regression)

Reply:
I've added a regression test: https://github.com/standardnotes/snjs/pull/12/commits/1f72faa785d896287263d17ec5e3657067bbaf1f